### DISCUSSION TITLE
不知道作者可否考虑自定义UA？

### DISCUSSION THREAD

**Discussion by keventseng** [tw93/Pake#281] [General]
打包网页后，像一些特定的页面应用场景希望能根据UA进行相关的数据控制以及页面上的一些特性调整。望作者能采纳

**Comment by keventseng** [comment]
翻看了一下旧的讨论，有发现作者说支持，但好像文档上没有发现，不知道在哪里配置？

**Reply by Tlntin** [reply]
pake-cli的dev分支目前支持自定义ua，master分支暂时未支持。
dev 分支体验
```bash
npm install https://github.com/tw93/Pake.git#dev -g
```
使用[文档](https://github.com/tw93/Pake/blob/dev/bin/README.md#user-agent)

**Reply by keventseng** [reply]
好的，已解决
